pkg/kubelet/kubelet.go

```diff
@@ -478,6 +480,7 @@ func NewMainKubelet(
 		return nil, err
 	}
 	klet.containerGC = containerGC
+	klet.containerDeletionWorker = newContainerDeletionWorker(klet.containerRuntime)
```

Comment:
The runtime implementation provides a way to delete a specific container through DeleteContainer. The worker here is kind of an optimization. I use the worker to represent a go routine that blocks on DeleteContainer calls. Another simpler approach is to scratch the worker idea and just do "go DeleteContainer(...)" in line 2358. Please advise.